split-webapp/split/src/App.css

```diff
@@ -40,9 +40,37 @@
   flex-direction: column;
   position: absolute;
   width: 17%;
+  min-width: 17%;
   height: 100%;
   left: 0;
   top: 0;
+  margin-top: 65px;
+}
+
+.CollapsedSidebar {
+  background-color: #79a2f4;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 3.5%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  margin-top: 65px;
+  min-width: 70px;
```

Comment:
Thanks for the feedback. It would definitely be good to avoid duplication here. Just to clarify, although there are many similarities, there are also key differences such as the width of the bar (i.e. thin bar when collapsed, thick bar when expanded). If I were to combine it into a shared css rule like you mentioned, how/where should I specify the different bar widths?

Reply:
One way would be to specify the differing CSS attributes in their own rules:

```css
.ClassA, .ClassB {
  /* ... shared attributes ... */
}
.ClassA {
  /* ... attributes for .ClassA only ... */
}
.ClassB {
  /* ... attributes for .ClassB only ... */
}
```